torch/_inductor/codecache.py

```diff
@@ -1764,6 +1773,17 @@ def _compile_consts_darwin(consts: bytes) -> str:
             )
 
             output_o = os.path.splitext(input_path)[0] + ".o"
+            consts_size = sum(
+                tensor.untyped_storage().nbytes()
+                for (name, tensor) in graph.constants.items()
+                if name not in graph.folded_constants
+            )
+            # TODO: Fix mmap weights with cuda
+            use_mmap_weights = (
+                not cuda and config.is_fbcode() and consts_size > 3_000_000_000
```

Comment:
`config.is_fbcode()` prevents this from being useful in OSS, but this is not an fbcode-specific issue, is it?

Reply:
Thank you for the thorough review. Inverted condition and reduced limit to 2e9

Reply:
why do we need fbcode gating at all? the issue should also affect fbcode, shouldn't it?

Reply:
Because I'm on PTO and have no access to fbcode to debug what might go wrong. But yes, we should remove it in the long run (though I thought long term plan is to get rid of weights bundling into `.so` altogether)